src/modules/components/proposal/proposalActions/proposalActions/proposalActions.test.tsx

```diff
@@ -9,6 +9,8 @@ import { ProposalActions, type IProposalActionsProps } from './proposalActions';
 jest.mock('../../../member', () => ({ MemberAvatar: () => <div data-testid="member-avatar" /> }));
 
 describe('<ProposalActions /> component', () => {
+    let scrollIntoViewSpy: jest.SpyInstance;
```

Comment:
Can't we create the spy as we're doing on other tests like:
```
const scrollIntoViewSpy = jest.spyOn(HTMLElement.pototype, 'scrollIntoView')

afterEach(() => {
  scrollIntoViewSpy.mockReset();
}
```

Reply:
any risk with doing something like this?

```
HTMLElement.prototype.scrollIntoView = jest.fn();
const scrollIntoViewSpy = jest.spyOn(HTMLElement.prototype, 'scrollIntoView');
 
...
 
afterEach(() => {
  scrollIntoViewSpy.mockReset();
}
```

I had it the previous way because scrollIntoView doesn't exist on the prototype unless set. @cgero-eth

Reply:
Yes makes sense, as it's not available on Jest environment I updated the test config to make it available if we need to use/mock it for other tests 👌🏻